src/types/global.d.ts

```diff
@@ -32,3 +32,7 @@ declare const after: typeof afterAll
 declare interface Array<T> {
   __last: () => T;
 }
+
+declare interface Function {
+  estimateGas(mainParams?: any, txParams?: any): any;
```

Comment:
Ugh, every function will have it like that.
Instead in api/types do
```
type DutchExchangeExtended = {
  [K in keyof DutchExchange]: DutchExchange[K] extends (...args: any[]) => Promise<Receipt> ?
    DutchExchange[K] & {sendTransaction?: DutchExchange<Hash>[K], estimateGas?: DutchExchange<Hash>[K]} :
    DutchExchange[K]
}
```
wherever sendTransaction appears add estimateGas